src/disasm_main.cpp

```diff
@@ -38,6 +38,7 @@
 #include "passes/FunctionInferencePass.h"
 #include "passes/NoReturnPass.h"
 #include "passes/SccPass.h"
+#include "DwarfMap.hpp"
```

Comment:
The other CPP header files have a `.h` extension, should change for consistency.